common4j/versioning/version.properties

```diff
@@ -0,0 +1,4 @@
+#Wed May 05 14:08:27 PDT 2021
+versionName=0.0.4
+versionCode=1
+latestPatchVersion=213
```

Comment:
Is this number valid?

Reply:
Hopefully, I will make sure to update it before I merge